### ISSUE TITLE
Redundant bias 

### ISSUE THREAD

**Issue by rwightman** [google/automl#65]
Picking through in some detail for a faithful PyTorch adaptation I noticed that there are some instances of back to back biases (conv w/ bias followed by batch norm layers). Was this simple oversight or done intentionally? 

![double_bias](https://user-images.githubusercontent.com/5702664/77703509-3853dc00-6f78-11ea-8866-97f328334afa.png)

**Comment by mingxingtan** [comment]
Oh, good point. that's oversight. The conv bias is probably not necessary if followed by batch norm layer.

**Comment by mingxingtan** [comment]
BTW, @rwightman , the previous code and checkpoints have some bugs. I have fixed the bugs and uploaded new checkpoints recently. Please do git pull and download the latest checkpoint. Sorry for the troubles.

**Comment by rwightman** [comment]
@mingxingtan  I noticed those changes coming in, thanks.

For this issue, you could likely disable the conv bias in code and process the checkpoint graphs to merge it into the BN beta...  it's a pretty minor issue either way. For inference graphs it's likely optimized out anyways.

**Comment by bitwangdan** [comment]
@rwightman 
hi, Can you share a graph of the model with me? Model is big, the tensorboard in my PC doesn't work

**Comment by rwightman** [comment]
@bitwangdan I used Tensorboard, it doesn't look like anything there, but it's because there are so many cond nodes spread out it zooms out to extremes. If you select/filter some known nodes by name and click you can zoom into a smaller area and then fumble around with the panning. It's pretty frustrating to navigate but it kinda works.

**Comment by mingxingtan** [comment]
@rwightman @bitwangdan Looks like you are viewing training graph? Yeah, that would be very painful.

I have just submitted https://github.com/google/automl/commit/9453a61a80477db40d47dea670b545362bb567b5 and https://github.com/google/automl/commit/0d137d003a9a282149f425a4a3deafca39cb7a84, so you can just run:

```
python model_inspect.py --model_name=efficientdet-d0 --logdir=/tmp/effdet0/
tensorboard --logdir=/tmp/effdet0/
```

Then you should be able to see a much nicer tensorboard like this:

<img width="728" alt="tensorboard" src="https://user-images.githubusercontent.com/6027221/77994527-09729880-72df-11ea-9ad6-1b6ba04e2892.png">

**Comment by mingxingtan** [comment]
In case you are not able to run tensorboard, I have also updated [tutorial.ipynb](https://colab.sandbox.google.com/github/google/automl/blob/master/efficientdet/tutorial.ipynb) to show the graph in tensorboard, so you can just run everything in colab.

**Comment by rwightman** [comment]
@mingxingtan yeah, I was looking at the training graph, I'll try with the new additions, thanks
